packages/crypto/tsconfig.json

```
@@ -3,5 +3,5 @@
   "compilerOptions": {
     "outDir": "./dist"
   },
-  "include": ["src"]
+  "include": ["src", "test/**/*"]
```

Nit - I think this would suffice.

```suggestion
  "include": ["src", "test"]
```